shared/domains/chelonia/internals.js

```diff
@@ -372,6 +372,9 @@ export default (sbp('sbp/selectors/register', {
           const response = await fetch(`${this.config.connectionURL}/time`, { signal: this.abortController.signal })
           return handleFetchResult('text')(response)
         } catch (e) {
+          if (fallback) {
+            return new Date(sbp('chelonia/time') * 1000).toISOString()
+          }
```

Comment:
I checked, and it seems that everywhere `fetchServerTime` is called you've decided to pass `true` for the `fallback`.

Should it be `true` by default then?

And what are the security considerations here? Should they perhaps be documented here in a comment?

Reply:
There are no security considerations because the time can be set to whatever the client wants to set it to. We're not using signed timestamps. The overarching security consideration (which hasn't changed with this modification) is that the timestamp is to be treated as suggestive, not as gospel.

> I checked, and it seems that everywhere fetchServerTime is called you've decided to pass true for the fallback.

Correct.

> Should it be true by default then?

I considered it, and we could. I decided against it because a function named `fetchServerTime` strongly suggests a fetch of some kind is being done. The change overrides the fetch in certain situations (OTOH, without this change, there's no way to use this function in contracts without being online).

Reply:
Think about this from the perspective of developers... do we want them to think about this?

Or should we just tell them to call this function and not worry about it?

I think it's OK to set it to `true` by default, especially because `'chelonia/time'` is technically the server time - at least it should be set to that based on the last time it sync'd.